examples/hello_world/build.rs

```diff
@@ -1,11 +1,12 @@
-extern crate protoc_rust;
+extern crate protoc_rust_grpc;
 
 fn main() {
-    protoc_rust::run(protoc_rust::Args {
+    protoc_rust_grpc::run(protoc_rust_grpc::Args {
```

Comment:
This means that we are using our forked version to generate both services and messages. This means that we will need to maintain this for both use cases. Can we use the default version for the services and use the fork only for the "custom code"?